themes/default/content/docs/intro/console/organizations.md

```diff
@@ -29,6 +30,8 @@ Organizations are a space for you to collaborate on shared projects and stacks.
 
 ## Creating an Organization
 
+Every organization has it's own subscription.
```

Comment:
I feel like we probably could put this in-context in the bullet list below as well or instead. 

I'm also not sure "subscription" is a term we use anywhere, so it may not be clear what this means.

I expect we want to describe a little more fully as part of the "start a free trial" note below that each organization will start a trial of Enterprise (linking to pricing page or product details page), and can choose to move to Team or Enterprise before their trial completes.  Then in that context, noting that each organization is on a separate plan.

Reply:
we still use subscription in the console, but its been mostly removed from the website and replaced with edition, which is kinda funky to use in a sentence, but ill give it a try.

Reply:
@lukehoban i tried again, let me know what you think. id prefer to keep the list of instructional steps short and not too complicated. but if you see a way to add it in there pls let me know.